src/main/java/com/openshift/internal/restclient/ResourceFactory.java

```diff
@@ -104,7 +104,7 @@
 	}
 	private IClient client;
 	
-	public ResourceFactory(IClient client) {
+	protected ResourceFactory(IClient client) {
```

Comment:
I was thinking that we wouldnt want anyone outside to be able to instantiate the factory. Final would be something different, it was disallow subclassing. Dont agree?